sycl/include/CL/sycl/detail/pi.hpp

```diff
@@ -359,11 +359,23 @@ class DeviceBinaryImage {
     return Format;
   }
 
-  /// Gets the iterator range over specialization constants in this this binary
-  /// image. For each property pointed to by an iterator within the range, the
-  /// name of the property is the specializaion constant symbolic ID and the
-  /// value is 32-bit unsigned integer ID.
-  const PropertyRange &getSpecConstants() const { return SpecConstIDMap; }
+  /// Gets the iterator range over scalar specialization constants in this this
```

Comment:
```suggestion
  /// Gets the iterator range over scalar specialization constants in this
```

Reply:
Fixed in 0800c2bc8278c364eeee5951d0be180df228dcef